mldsa/native/aarch64/src/arith_native_aarch64.h

```diff
@@ -27,6 +27,13 @@ extern const int32_t mld_aarch64_intt_zetas_layer78[];
 extern const int32_t mld_aarch64_intt_zetas_layer123456[];
 
 extern const uint8_t mld_rej_uniform_table[];
+extern const uint8_t mld_rej_uniform_eta_table[];
+
+#define MLD_AARCH64_REJ_UNIFORM_ETA2_BUFLEN \
+  (1 * 136) /* POLY_UNIFORM_ETA_NBLOCKS * SHAKE256_RATE */
```

Comment:
Typo: POLY_UNIFORM_ETA2_NBLOCKS? How is this calculated?

Reply:
I added comments explaining how these numbers came about. 
For eta=2, the parameter appears to be selected heuristically as one expects to need 136.5 bytes. It depends on the relative performance of Keccak x1 and Keccak x4 whether it makes sense to sample 1 or 2 blocks initially. Overall it does not appear to matter much.

Reply:
Thanks!